synthetic_data/synthetic_data.py

```diff
@@ -206,7 +219,7 @@ def make_tabular_data(
         p_thresh - probability threshold for assigning class labels
         noise_level_x (float) - level of white noise (jitter) added to x
         noise_level_y (float) - level of white noise added to y (think flip_y)
-        feature_range (list) - list of lower and upper range for feature scaling (e.g., [0,1]).
+        scaler (sklearn scaler) - sklearn style scaler. Defaults to MinMaxScaler(feature_range = (0,1)).
```

Comment:
Looks like the default is (-1, 1)

Reply:
Yup - for clean synthetic data with no outliers, the initial design consideration was to have only 1 scaler, so the range [-1,1] was the most straightforward.  And should probably remain the default.

Reply:
Right, need to change the value in the docstring which states (0, 1)